pkg/newlog/cmd/newlogd.go

```diff
@@ -254,6 +231,7 @@ func main() {
 	if err != nil {
 		log.Fatal(err)
 	}
+	// Activate calls populate() internally to retrieve persistent data (if any) and call handlers
```

Comment:
How about adding
 We thus assume that once Acrivate returns the handleGlobalConfigCreate has been called.